airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala

```diff
@@ -386,7 +397,12 @@ object LogicalPlan {
       columnNames match {
         case Some(aliases) =>
           query.outputAttributes.zip(aliases).map { case (in, alias) =>
-            SingleColumn(in, Some(alias))
+            SingleColumn(
+              in,
+              Some(alias),
+              None,
+              alias.nodeLocation // TODO Is alias.nodeLocation suitable as NodeLocation for this?
```

Comment:
🤔

Reply:
It's not accurate, but returning the close position would be good enough.